apps/synapse-portal-framework/src/components/elportal/ELGettingStarted.tsx

```diff
@@ -0,0 +1,162 @@
+import { Box, Button, Typography, darken } from '@mui/material'
+import React from 'react'
+import exploreIcon from './assets/explore_icon.png'
+import uncoverIcon from './assets/uncover_icon.png'
+import accessIcon from './assets/access_icon.png'
+
+// TODO Fix vertical column space (getting started section is slightly bigger)
+// TODO Set padding for mobile and desktop
+
+const ELGettingStarted: React.FC = () => {
+  return (
+    <Box
+      sx={{
+        backgroundColor: '#39AC97',
+        display: 'grid',
+        gridTemplateColumns: '25% 25% 25% 25%',
+        position: 'relative',
+        pt: '80px',
+        pb: '80px',
+        pl: '80px',
+        pr: '80px',
+      }}
+    >
+      <Box
+        sx={{
+          flex: '1 1 auto',
+          width: '100%',
+          height: '100%',
+          borderTop: '3px solid #ffffff88',
+          pt: '20px',
+          pb: '20px',
+        }}
+      >
+        <Typography
+          variant="headline1"
+          style={{
+            color: 'white',
+            fontFamily: "'Merriweather', serif",
+            fontWeight: 400,
+            fontSize: '24px',
+          }}
+          sx={{
+            width: '100%',
+          }}
+        >
+          Getting Started
+        </Typography>
+        <Typography
+          variant="body1"
+          style={{
+            color: 'white',
+            fontStyle: 'italic',
+            fontSize: '13px',
+            lineHeight: '20px',
+          }}
+          sx={{
+            maxWidth: '100%',
+            mt: '16px',
+            mb: '16px',
+          }}
+        >
+          We provide all the help you need for navigating the portal and
+          accelerating your research.
+        </Typography>
+        <Button
+          variant="contained"
+          href="https://help.eliteportal.org/help/"
+          style={{
+            fontSize: '14px',
+            fontWeight: 500,
+          }}
+          sx={{
+            backgroundColor: '#39AC97',
+            border: '1px solid white',
+            padding: '6px 24px',
+            boxShadow: 'none',
+            '&:hover': {
+              backgroundColor: darken('#5BA998', 0.05),
+              boxShadow: 'none',
+            },
+          }}
+        >
+          Visit Our Help Section
+        </Button>
```

Comment:
Probably don't need some of these styles -- the default button color is `primary.main`, and it handles hover/active states for us. 

Nice catch to remove the box shadow though, that would look weird here! Normally I would never advise to use `!important` but since it's only applied to one element, I think you could get away with it. This would handle the default, hover, and active states of the button.


```suggestion
        <Button
          variant="contained"
          href="https://help.eliteportal.org/help/"
          sx={{
            border: '1px solid white',
            padding: '6px 24px',
            boxShadow: 'none !important',
          }}
        >
          Visit Our Help Section
        </Button>
```